package.json

```diff
@@ -253,7 +253,8 @@
     "ts-mixer@npm:^6.0.3": "patch:ts-mixer@npm%3A6.0.4#~/.yarn/patches/ts-mixer-npm-6.0.4-5d9747bdf5.patch",
     "sucrase@npm:3.34.0": "^3.35.0",
     "@expo/config/glob": "^10.3.10",
-    "@expo/config-plugins/glob": "^10.3.10"
+    "@expo/config-plugins/glob": "^10.3.10",
+    "@npmcli/run-script/node-gyp": "9.3.0"
```

Comment:
@montelaidev I think we want the opposite of this? Why v9?

Reply:
Sorry, youre right, i misread the previous comment.